connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/operate/OperateClientConfiguration.java

```diff
@@ -14,12 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.connector.api.inbound;
+package io.camunda.connector.runtime.inbound.operate;
 
-/**
- * Defines the behavior for a connector that performs polling operations.
- *
- * <p>This interface extends {@link InboundConnectorExecutable} to specify functionality for
- * connectors that periodically query an external system to fetch information or detect changes.
- */
-public interface PollingConnectorExecutable extends InboundConnectorExecutable {}
+import io.camunda.connector.runtime.core.inbound.OperateClientAdapter;
+import io.camunda.operate.CamundaOperateClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OperateClientConfiguration {
+  @Bean
+  public OperateClientAdapter springOperateClientAdapter(
+      CamundaOperateClient camundaOperateClient) {
+    return new OperateClientAdapterImpl(camundaOperateClient);
+  }
+}
```

Comment:
I might be wrong, but it seems that this bean method can fail if inbound connectors are disabled by setting `camunda.connector.polling.enabled=false`, and Operate client bean is not created.

Reply:
Sorry, false alarm - as discussed with @Oleksiivanov this is actually safe, because `OperateClientConfiguration` is imported in the `InboundConnectorRuntimeConfiguration`, which has the `ConditionalOnProperty` annotation.